website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md

```diff
@@ -7,71 +7,112 @@ sidebar_label: "Connecting GitLab"
 ## Overview
 
 Connecting your GitLab account to dbt Cloud unlocks exciting and compelling functionality in dbt Cloud. Once your GitLab account is connected, you can:
-- [Trigger CI builds](cloud-enabling-continuous-integration-with-github) when Merge Requests are opened in GitLab.
-- Import new repos with one click
-- Carry GitLab user permissions through to dbt Cloud IDE's git actions.
-## For Developer and Team tiers
+- Log into dbt Cloud via OAuth through GitLab.
+- Import new GitLab repos with a couple clicks during dbt Cloud project setup.
+- Clone repos using HTTPS rather than SSH.
+- Trigger [Continuous integration](cloud-enabling-continuous-integration-with-github) builds when merge requests are opened in GitLab.
 
-To connect your personal GitLab account, navigate to your [Profile](https://cloud.getdbt.com/#/profile/) and select [Integrations](https://cloud.getdbt.com/#/profile/integrations/) from the left sidebar. 
+The steps to integrate GitLab in dbt Cloud depends on your plan. 
+If you are on:
+- the Developer or Team plan, read these [instructions](/connecting-gitlab#for-dbt-cloud-developer-and-team-tiers).
+- the Enterprise plan, jump ahead to these [instructions](/connecting-gitlab#for-the-dbt-cloud-enterprise-tier).
 
-If your account is not connected, click the button `Link your GitLab account` to continue the setup process. 
-<Lightbox src="/img/docs/dbt-cloud/connecting-gitlab/not-connected.png" title="Link your GitLab" />
+## For dbt Cloud Developer and Team tiers
```

Comment:
@runleonarun just flagging a potential usecase for `<tier/>` blocks similar to `<version/>` blocks. not sure how often this comes up though

Reply:
@dataders I have been thinking of how to handle this situation in the UI, but I think you are spot on that we'll need to deal with this use case.